scripts/tests/chiptest/test_definition.py

```diff
@@ -37,13 +37,14 @@ def __init__(self, runner, command):
         self.cv_stopped = threading.Condition()
         self.stopped = False
         self.lastLogIndex = 0
+        self.kvs = '/tmp/chip_kvs'
 
-    def start(self, discriminator):
+    def start(self, options):
```

Comment:
Use None as a default value for options (not empty dict since that is evaluated only once) and handle it or use `**options` for an implied dictionary.

Goal is to not require start without options to pass in an empty `{}`

Reply:
Thanks for the suggestion, updated the PR with the optional param